### PR TITLE
make clear we are not secure yet

### PR DESCRIPTION
**Pull request by tedbow**
We have https://github.com/php-tuf/php-tuf/issues/25 to define security policies but in the meantime we should be extra clear this library should not yet be considered secure or be used in production

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # PHP-TUF
 
+## IMPORTANT
+PHP-TUF is in a pre-release state is not considered a complete or secure version of the TUF framework.
```
Comment (effulgentsia) [inline README.md:4]: s/state is/state and is/

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+It should only currently be used for testing, development and feedback.
```
Comment (effulgentsia) [inline README.md:5]: s/only currently/currently only/

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+## IMPORTANT
+PHP-TUF is in a pre-release state is not considered a complete or secure version of the TUF framework.
```
Comment (effulgentsia) [inline SECURITY.md:2]: s/state is/state and is/

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+It should only currently be used for testing, development and feedback.
```
Comment (effulgentsia) [inline SECURITY.md:3]: s/only currently/currently only/

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+*Do not use in production for secure target downloads!!*
+
+We appreciate any help finding security issues.
+
+Security issues can be disclosed publicly until the first release. 
+If you think you have found a security issue please:
```
Comment (effulgentsia) [inline SECURITY.md:10]: s/issue please/issue, please/

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+1. Please search the [existing security issues](https://github.com/php-tuf/php-tuf/issues?q=is%3Aissue+is%3Aopen+security) to see if the issue is already known.
```
Comment (effulgentsia) [inline SECURITY.md:11]: s/Please search/Search/

**Review by effulgentsia — COMMENTED** [review]
+1 to the idea of this! Just some minor nits:

File: README.md (hunk 1 above)
Comment (tedbow) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (tedbow) [reply README.md]: fixed

File: SECURITY.md (hunk 3 above)
Comment (tedbow) [reply SECURITY.md]: fix

File: SECURITY.md (hunk 4 above)
Comment (tedbow) [reply SECURITY.md]: fixed

File: SECURITY.md (hunk 5 above)
Comment (tedbow) [reply SECURITY.md]: fixed

File: SECURITY.md (hunk 6 above)
Comment (tedbow) [reply SECURITY.md]: fixed

**Review by effulgentsia — APPROVED** [review]
Looks great!

**Pull request merged**